client/src/components/Form/Elements/FormText.vue

```diff
@@ -82,31 +81,12 @@ export default {
             type: Array,
             default: null,
         },
-
-        // These last three are for handling special input cases only
-
-        optional: {
-            type: Boolean,
-            default: false,
-        },
-        model_class: {
-            type: String,
-            default: "",
-        },
-        data: {
-            type: Object, // ?
-            default: () => {},
-        },
     },
     data() {
-        const inputArea =
-            this.multiple &&
-            (["SelectTagParameter", "ColumnListParameter"].includes(this.model_class) || (this.options && this.data));
```

Comment:
this definition of inputArea is not equivalent to what you suggest below: `this.area || this.multiple` 
Is the alternative sufficient? If so, it'll explain why those props are useless, but I'm sure they historically had some point, right? What was it and do we have to maintain anything in that regard?

Reply:
So in this branch `inputArea` only provides additional conditions to restrict displaying the `datalist`. I don't understand why we would want to do that. In current dev this condition is used to process values coming in from specific input types:
https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/Form/Elements/parameters.js#L134
But in this branch the input values are processed with the `reduce` if they are `Arrays`, regardless of input type which is better.